tests/vc-generator/TestLinkedDataSignature.js

```diff
@@ -158,30 +151,18 @@ export class LinkedDataSignature extends LinkedDataProof {
     return proof;
   }
 
-  /**
-   * @param document - {object} To be signed.
-   * @param purpose - {ProofPurpose}.
-   * @param documentLoader - {function}.
-   * @param expansionMap - {function}.
-   *
-   * @param document.proof
-   * @returns {Promise<object>} Resolves with the created proof object.
-   */
   async updateProof({proof}) {
     // extending classes may do more
     return proof;
   }
 
   /**
-   * @param proof.proof
-   * @param proof - {object} The proof to be verified.
-   * @param document - {object} The document the proof applies to.
-   * @param documentLoader - {function}.
-   * @param expansionMap - {function}.
+   * @param {object} options - Options to use.
+   * @param {object} options.proof - The proof to be verified.
+   * @param {object} options.document - The document the proof applies to.
+   * @param {Function} options.documentLoader - A JSON-LD document loader.
+   * @param {Function} options.expansionMap - A JSON-LD expansion map.
    *
-   * @param proof.document
-   * @param proof.documentLoader
-   * @param proof.expansionMap
    * @returns {Promise<{object}>} Resolves with the verification result.
    */
   async verifyProof({proof, document, documentLoader, expansionMap}) {
```

Comment:
`expansionMap` could be removed here and anywhere else.

Reply:
expansionMap removed here: https://github.com/w3c-ccg/di-ed25519-test-suite/pull/32/commits/202312f0bd378638bc38281b278fbbceb5d74d7a